go.mod

```diff
@@ -16,27 +16,19 @@ require (
 	github.com/magiconair/properties v1.7.4 // indirect
 	github.com/mitchellh/mapstructure v0.0.0-20180111000720-b4575eea38cc // indirect
 	github.com/onsi/ginkgo v1.10.2
-	github.com/onsi/gomega v1.5.0
+	github.com/onsi/gomega v1.7.0
 	github.com/pelletier/go-toml v1.0.1 // indirect
-	github.com/spf13/afero v1.2.2 // indirect
 	github.com/spf13/cast v1.1.0 // indirect
 	github.com/spf13/cobra v0.0.3
 	github.com/spf13/jwalterweatherman v0.0.0-20180109140146-7c0cea34c8ec // indirect
-	github.com/spf13/pflag v1.0.3
+	github.com/spf13/pflag v1.0.5
 	github.com/spf13/viper v1.0.0
-	github.com/stretchr/testify v1.4.0 // indirect
 	github.com/wadey/gocovmerge v0.0.0-20160331181800-b5bfa59ec0ad // indirect
-	istio.io/api v0.0.0-20191115173247-e1a1952e5b81
-	istio.io/client-go v0.0.0-20191120150049-26c62a04cdbc
-	k8s.io/api v0.0.0-20191025225708-5524a3672fbb
-	k8s.io/apimachinery v0.0.0-20191025225532-af6325b3a843
-	k8s.io/client-go v0.0.0-20190620085101-78d2af792bab
+	istio.io/api v0.0.0-20200316191724-7632cba5ff4b
+	istio.io/client-go v0.0.0-20200316192452-065c59267750
+	k8s.io/api v0.17.0
```

Comment:
I'm seeing 0.17.2 for the k8s api [here](https://github.com/istio/istio/blob/release-1.5.1-patch/go.mod) and on the 1.5 branch as well. Do we want to update it to the version that istio is using?

Reply:
I didn't update any of the k8s dependencies either. k8s 0.17.0 is what istio is using in 1.5.1. My one `go get ...` updated all of this (I did manually remove the `replace` bits).